map.css

```diff
@@ -2,3 +2,79 @@
   height: 40vw;
   width: 100%;
 }
+
+#searchMap {
+  height: 40vw;
```

Comment:
again, for height we relate to height of viewport (vh) - there is *no* correlation between height and width of window so it's very unsafe to relate to the other property